.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -107,27 +105,41 @@ assignees: ''
 
         * redo step 8, but now targetting the latest payload image generated for Kata Containers and Enclave CC
 
-- [ ] - 18. Make sure all the operator tests are passing
+- [ ] - 19. Make sure all the operator tests are passing
 
 - [ ] - 19. Cut an Enclave CC release
 
 - [ ] - 20. Add a new Kata Containers tag
 
+
+- [ ] - 21. Wait for release kata-runtime-payload to be successfully built
+        * After the Kata tag is created wait for (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload.yaml) to be successfully completed, so the latest commit kata-runtime-payload for the release is created
+
+- [ ] - 22. Contact @stevenhorsman and/or @bpradipt to get the peer pods release ready
+  - Update the versions of the peer pods dependencies:
+    - attestation-agent - set to the the release tag
+    - kata-containers - set to the release tag
+  - Create the new peer pods  release: 
```

Comment:
extract space pods\s\srelease

Reply:
BTW, I just created an issue (https://github.com/confidential-containers/cloud-api-adaptor/issues/1108) suggesting that we have a release checklist on peer pods as well. We could have the content of this bullet detailed there, and then just link here.

Reply:
I'm going to suggest waiting for a day or so to merge this and I'll try and move/write the release process in the cloud-api-adaptor and then link to that as suggested.

Reply:
Works for me.

Reply:
I've created https://github.com/confidential-containers/cloud-api-adaptor/pull/1110 for this, so once it gets reviewed and merged, we can link to sections in there.